Collaboration (#142)
XMPro/Blueprints-Accelerators-Patterns: docs/collections/_blueprints/general-ra-meta-tags.md

gallery:

>   - image: "/assets/images/blueprints/general-ra-meta-tags/ad03.png"

Reviewer: ```suggestion
  - image: "/assets/images/blueprints/general-aa-meta-tags/ad03.png"
```

Author: done

    caption: "<strong>Above:</strong> Application View"

>   - image: "/assets/images/blueprints/general-ra-meta-tags/ad04.png"

Reviewer: ```suggestion
  - image: "/assets/images/blueprints/general-aa-meta-tags/ad04.png"
```

    caption: "<strong>Above:</strong> Reference Table View"
gallery_limit: 2

files:
  - title: "Application"

>     link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/application"

Reviewer: ```suggestion
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-aa-meta-tags/application"
```

Author: done

    image: "/assets/images/icons/AD.png"
  - title: "SQL"
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/sql"
    image: "/assets/images/icons/sql.webp"
  - title: "Datastream"
    link: "https://github.com/XMPro/Blueprints-Accelerators-Patterns/blob/master/blueprints/general-ra-meta-tags/datastream"
    image: "/assets/images/icons/DS.png"

---

## Application

> This blueprint app establishes meta tag labels for recommendations. This allows historical recommendations to be filtered and analysed by labels such as asset class, location, training recommendations, or area. Intended uses could include:

Reviewer: ```suggestion
This Blueprint app establishes Meta Tag labels for recommendations. This allows historical recommendations to be filtered and analysed by labels such as asset class, location, training recommendations, or area. Intended uses could include:
```

Author: done

- Recommendation Analytics;
- Filters available for 'Suspend Recommendations for shutdown';
- Threshold comparisons by comparing various recommendations by asset class.

### Create Meta Tags

<div class="inline_image">{% include framework/shortcodes/image.html src="/assets/images/blueprints/general-ra-meta-tags/ad01.png" %}</div>

> This page contains a navigation bar at the top right and a recommendation selection grid to create/ update the meta tag. 

Reviewer: ```suggestion
This page contains a navigation bar at the top right and a recommendation selection grid to create/update the Meta Tag. 
```

Author: done

> Each grid has the Store User option enabled meaning it will remember your selection for column ordering, grouping, and filtering.  

Reviewer: ```suggestion
Each grid has the Store User option enabled, which means it will remember your selections for column ordering, grouping, and filtering.  
```

Author: done
